src/common/actions/select-repositories-form.js

```diff
@@ -0,0 +1,8 @@
+export const TOGGLE_REPOSITORY = 'SELECT_REPOSITORY_FORM_TOGGLE_REPOSITORY';
```

Comment:
The reason the value is not the same as the key is that I expect "TOGGLE_REPOSITORY" to one day be used in more than one place, so I am namespacing the action value here so that it doesn't clash if we do add similar action elsewhere.